dockerFiles/marklogic-server-centos:base

```diff
@@ -46,7 +46,9 @@ ENV MARKLOGIC_INSTALL_DIR=/opt/MarkLogic  \
     MARKLOGIC_VERSION="${ML_VERSION}" \
     MARKLOGIC_BOOTSTRAP_HOST=bootstrap \
     MARKLOGIC_ADMIN_USERNAME_FILE=mldb_admin_user \
-    MARKLOGIC_ADMIN_PASSWORD_FILE=mldb_password_user
+    MARKLOGIC_ADMIN_PASSWORD_FILE=mldb_password_user \
+    MARKLOGIC_LICENSE=mldb_liense \
```

Comment:
I would recommend this be "LICENSE_KEY" to match the wording in the admin UI as well as the docs https://docs.marklogic.com/REST/POST/admin/v1/init

Reply:
Ok I've updated it. 
aeecdd9

I also changed the licensee to be `LICENSEE` instead to be inline with the rest api doc.